README.md

```diff
@@ -19,17 +27,52 @@ There are different optional dependencies for SAM, if you are unsure use `pip in
 
 Keep in mind that the sam package is updated frequently, and after a while, your local version may be out of date with the online documentation. To be sure, run the `pip install -U sam` command to install the latest version.
 
-## Configuration
+### Simple example
 
-A configuration file can be created as `.config`. This configuration file only stores api credentials for now, but more options may be added in the future. The configuration file is parsed using the [Python3 configparser](https://docs.python.org/3/library/configparser.html), and an example configuration is shown below:
+Below you can find a simple example on how to use one of our timeseries models. For more examples, check our [example notebooks](https://github.com/RoyalHaskoningDHV/sam/tree/main/examples)
+
+```python
+import pandas as pd
+from sam.models import MLPTimeseriesRegressor
+from sam.feature_engineering import SimpleFeatureEngineer
+
+data = pd.read_parquet("../data/rainbow_beach.parquet").set_index("TIME")
```

Comment:
`load_rainbow_beach()`  😁

Reply:
Will do that in a new PR, since this function is not yet in the main branch